<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multifunctool</class>
 <widget class="QMainWindow" name="multifunctool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <widget class="QTableWidget" name="tw9"/>
        </item>
        <item row="0" column="1">
         <widget class="QTableWidget" name="tw1">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="9">
         <widget class="QTableWidget" name="tw5"/>
        </item>
        <item row="0" column="8">
         <spacer name="horizontalSpacer_21">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="12">
         <spacer name="horizontalSpacer_22">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="7">
         <widget class="QTableWidget" name="tw10"/>
        </item>
        <item row="0" column="10">
         <spacer name="horizontalSpacer_23">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="11">
         <widget class="QTableWidget" name="tw6"/>
        </item>
        <item row="1" column="9">
         <widget class="QTableWidget" name="tw11"/>
        </item>
        <item row="1" column="11">
         <widget class="QTableWidget" name="tw12"/>
        </item>
        <item row="0" column="6">
         <spacer name="horizontalSpacer_24">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="7">
         <widget class="QTableWidget" name="tw4"/>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tw7"/>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_25">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QTableWidget" name="tw8"/>
        </item>
        <item row="0" column="5">
         <widget class="QTableWidget" name="tw3"/>
        </item>
        <item row="0" column="3">
         <widget class="QTableWidget" name="tw2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,20,0,0,0,0,1,1">
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>已连接测试设备MAC地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="mac_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_12" columnstretch="1,5">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>8</number>
            </property>
            <item row="4" column="1">
             <widget class="QTextEdit" name="wifi_pwd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="wifi_ssid">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>30</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_16">
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>5</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Wifi密码</string>
              </property>
              <property name="margin">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="text">
               <string>Wifi名称</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="margin">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>请输入需要在测试时连接的Wifi名称及密码</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_13" columnstretch="1,0,0">
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_26">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_27">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="devNum">
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="connCom"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_19">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>请选择网关个数</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_20">
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>请选择上电端口</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>维修模式测试选项</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_14" rowstretch="0,0,0,0" columnstretch="0,0">
            <item row="2" column="0">
             <widget class="QPushButton" name="wtd_test">
              <property name="text">
               <string>看门狗</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="usb_test">
              <property name="text">
               <string>USB</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="wifi_test">
              <property name="text">
               <string>Wifi</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="eth_test">
              <property name="text">
               <string>以太网</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="fourG_test">
              <property name="text">
               <string>4G</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="rtc_test">
              <property name="text">
               <string>RTC</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="3" column="0">
             <widget class="QPushButton" name="connect_btn">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="mod_btn">
              <property name="text">
               <string>维修模式</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="close_btn">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>断开</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="get_ip_btn">
              <property name="text">
               <string>仅获取IP</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="auto_test_btn">
              <property name="text">
               <string>测试模式</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="exit_btn">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="load_img_btn">
              <property name="text">
               <string>烧录</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked()</signal>
   <receiver>multifunctool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1140</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>1136</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
